packages/react-composites/src/composites/localization/locales/de-DE/strings.json

```diff
@@ -107,4 +107,4 @@
     "openDtmfDialpadLabel": "Wähltastatur anzeigen",
     "dtmfDialpadPlaceHolderText": "Nummer eingeben"
   }
-}
\ No newline at end of file
+}

```

Comment:
Assume these were added by a code editor, they should be removed from this changelist - then file a bug to standardize and lint these files

Reply:
![image](https://user-images.githubusercontent.com/11863655/199130768-e07ca195-a732-4d58-957f-34a89bfdd867.png)
Looks like it is running when you are doing a 'git commit' - how do we turn it off?

Reply:
Found it in precommit config file